x/fbridge/keeper/transfer.go

```diff
@@ -23,7 +23,7 @@ func (k Keeper) handleBridgeTransfer(ctx sdk.Context, sender sdk.AccAddress, amo
 	}
 
 	if err := k.bankKeeper.SendCoinsFromAccountToModule(ctx, sender, types.ModuleName, token); err != nil {
-		panic(err)
+		return 0, err
 	}
 
 	if err := k.bankKeeper.BurnCoins(ctx, types.ModuleName, token); err != nil {
```

Comment:
How about changing this case as well?

From:
```go
if err := k.bankKeeper.BurnCoins(ctx, types.ModuleName, token); err != nil {
    panic(fmt.Errorf("cannot burn coins after a successful send to a module account: %v", err))
```
To: 
> you can use %w to wrap another error
```go
if err := k.bankKeeper.BurnCoins(ctx, types.ModuleName, token); err != nil {
    return 0, fmt.Errorf("cannot burn coins after a successful send to a module account: %w", err)
```

Reply:
this case must not be executed and cannot be triggered by client. So I think it's better to keep it.

Reply:
Okay.